### ISSUE TITLE
pip install broken?

### ISSUE THREAD

**Issue by colinmorris** [mediawiki-utilities/python-mwviews#18]
(Apologies if this is just a quirk of my local setup, but I noticed there was a recent change to the way the project is packaged, and thought that might be related to the issue I'm having.)

I ran `pip install mwviews` and it appeared to install successfully. But if I try to run something like `python -m mwviews`, I get `No module named mwviews`.

`pip show mwviews` shows the package as being installed. Its location is given as: `Location: /home/colin/.local/lib/python3.6/site-packages` (as expected)

However, my site-packages does *not* have a `mwviews` subdirectory. And if I try to use pip to uninstall mwviews, the list of files to be removed looks like...

```
  /home/colin/.local/bin/mwviews
  /home/colin/.local/lib/python3.6/site-packages/api/__init__.py
  /home/colin/.local/lib/python3.6/site-packages/api/__pycache__/__init__.cpython-36.pyc
  /home/colin/.local/lib/python3.6/site-packages/api/__pycache__/pageviews.cpython-36.pyc
  /home/colin/.local/lib/python3.6/site-packages/api/pageviews.py
  /home/colin/.local/lib/python3.6/site-packages/mwviews-0.2.0.dist-info/INSTALLER
  /home/colin/.local/lib/python3.6/site-packages/mwviews-0.2.0.dist-info/LICENSE.txt
  /home/colin/.local/lib/python3.6/site-packages/mwviews-0.2.0.dist-info/METADATA
  /home/colin/.local/lib/python3.6/site-packages/mwviews-0.2.0.dist-info/RECORD
  /home/colin/.local/lib/python3.6/site-packages/mwviews-0.2.0.dist-info/WHEEL
  /home/colin/.local/lib/python3.6/site-packages/mwviews-0.2.0.dist-info/entry_points.txt
  /home/colin/.local/lib/python3.6/site-packages/mwviews-0.2.0.dist-info/top_level.txt
  /home/colin/.local/lib/python3.6/site-packages/utilities/__init__.py
  /home/colin/.local/lib/python3.6/site-packages/utilities/__pycache__/__init__.cpython-36.pyc
  /home/colin/.local/lib/python3.6/site-packages/utilities/__pycache__/aggregate.cpython-36.pyc
  /home/colin/.local/lib/python3.6/site-packages/utilities/__pycache__/fetch_global_namespaces.cpython-36.pyc
  /home/colin/.local/lib/python3.6/site-packages/utilities/__pycache__/util.cpython-36.pyc
  /home/colin/.local/lib/python3.6/site-packages/utilities/aggregate.py
  /home/colin/.local/lib/python3.6/site-packages/utilities/fetch_global_namespaces.py
  /home/colin/.local/lib/python3.6/site-packages/utilities/util.py
```

In other words, it looks to me like the contents of the package is getting exploded into the top-level `site-packages` directing, rather than its own proper subdirectory. (In fact, it turns out that as a workaround I can do something like `from api import PageviewsClient` rather than `from mwviews.api import PageviewsClient`)

**Comment by milimetric** [comment]
this indeed seems like broken packaging.  Thanks for noticing, I'll see
what I did wrong and try to fix!

**Comment by milimetric** [comment]
Ok, I published 0.2.1 on pypi, please give that a shot and let me know if the install has issues.  I noticed some of the commands on the executable aren't working anymore, but hopefully the general package is ok now.

**Comment by colinmorris** [comment]
Seems to be fixed with 0.2.1. Thanks!
